HPCC-9691 DOCS:Handling removal thor nodes

g-pan opened the pull request:
FIX HPCC-9691 DOCS:Handling removal thor nodes
Added section Redefining nodes in a Thor Cluster
(Ch. 6 Sect. 6) To Systems Admin doc.

Signed-off-by: G Panagiotatos greg.panagiotatos@lexisnexis.com

@jakesmith @JamesDeFabia please review

g-pan committed:
HPCC-9691 DOCS:Handling removal thor nodes

FIX HPCC-9691 DOCS:Handling removal thor nodes
Added section Redefining nodes in a Thor Cluster
(Ch. 6 Sect. 6) To Systems Admin doc.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

JamesDeFabia commented on docs/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml:
```diff
@@ -1195,7 +1195,50 @@ lock=/var/lock/HPCCSystems</programlisting>
+
+      <para>There are a couple of scenarios: <orderedlist>
+          <listitem>
+            <para>Replacing faulty node(s) If data files are replicated,
```
needs some punctuation

JamesDeFabia commented on docs/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml:
```diff
@@ -1195,7 +1195,50 @@ lock=/var/lock/HPCCSystems</programlisting>
+          </listitem>
+
+          <listitem>
+            <para>Resizing the cluster If you are adding or removing Thor
```
needs some punctuation

JamesDeFabia commented on docs/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml:
```diff
@@ -1195,7 +1195,50 @@ lock=/var/lock/HPCCSystems</programlisting>
+            use the old group structure, while new files use the new group
+            structure.</para>
+          </listitem>
+        </orderedlist>In summary, if the environment changes it must be
```
In summary, if the Thor cluster changes it must be updated in the Dali.

g-pan committed:
HPCC-9691 DOCS:Handling removal thor nodes

FIX HPCC-9691 DOCS:Handling removal thor nodes
Added section Redefining nodes in a Thor Cluster
(Ch. 6 Sect. 6) To Systems Admin doc.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

g-pan committed:
HPCC-9691 DOCS:Handling removal thor nodes

FIX HPCC-9691 DOCS:Handling removal thor nodes
Added section Redefining nodes in a Thor Cluster
(Ch. 6 Sect. 6) To Systems Admin doc.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

JamesDeFabia commented:
Looks good to me now, but let @jakesmith have the final say

jakesmith commented:
Looks good to me.
